src/plugins/persistence/couch/scripts/lockObjects.mjs

```diff
@@ -0,0 +1,187 @@
+import debounce from 'p-debounce';
+import {Sema} from 'async-sema';
+import { parseArgs } from 'util';
+
+const {
+  values: { couchUrl, database, lock, unlock, startObjectKeystring, user, pass }
+} = parseArgs({
+  options: {
+    couchUrl: {
+      type: 'string',
+      default: 'http://127.0.0.1:5984'
```

Comment:
pull this out into a VARIABLE which matches the viper-couchdb and deployctl/actions/config

Reply:
Both couch URL and DB name now default to the environment variables used by deployctl